linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/bash/shared.sh

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_ubuntu
+# platform = multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
```

Comment:
```suggestion
# platform = multi_platform_all
```

Reply:
Good. I will include in the next update.

Reply:
Good. I will include in the next update.